lightspeed/modules/proc_view-content-parser-output.adoc 

```diff
@@ -0,0 +1,35 @@
+:_content-type: PROCEDURE
+
+[id="view-content-parser-output_{context}"]
+
+= Viewing the generated training data set
+
+After {AnsibleContentParser} scans your Ansible files, it generates the training data set in an output subdirectory within your local directory. The training data set includes a *ftdata.jsonl* file, which is the main output of the {AnsibleContentParser}. The file is available in JSON Lines files format, where each line entry represents a JSON object. You must upload this JSONL file to {ibmwatsonxcodeassistant} for creating a custom model. 
```

Comment:
Existing: "You must upload this JSONL file to {ibmwatsonxcodeassistant} for creating a custom model."

Proposed: "You must upload this JSONL file to {ibmwatsonxcodeassistant} to create a custom model."

Reply:
Done.